### PR TITLE
[PATCH] net: phylink: sfp: Add quirk for FINISAR 10G modules

### PR DESCRIPTION
**Pull request by Mickey201**
Finisar FTLF8536P4BCL and FTLX1471D3BCL can operate
in 1000Base-x mode as well
Tested on Accton AS5114 platform.

Signed-off-by: Mickey Rachamim <mickeyr@marvell.com>

### CODE REVIEW

**Comment by Mickey201** [comment]
As for the failed "DentOS Verify" failure.
According to the Jenkins report console:
https://jenkins.dent.dev/job/dentos-verify-main/90/console

One of the build commands failed with the below message:
15:14:46 ---> dentos-build.sh
15:14:46 + sudo apt-get install -y binfmt-support
15:14:46 **sudo: apt-get: command not found**
15:14:46 Build step 'Execute shell' marked build as failure

Not fully clear to us why it happens... @sonoble - may you please restart the build process to confirm it was not just a specific server issue?

**Comment by taraschornyiplv** [comment]
recheck

**Comment by taraschornyiplv** [comment]
For some DentOS verify was executed on centos build host rather than Ubuntu/Debian.

**Comment by keanjapesan** [comment]
recheck

**Review by taraschornyiplv — DISMISSED** [review]

**Comment by keanjapesan** [comment]
recheck

**Comment by sonoble** [comment]
The wrong OS is being used. Apt is failing.

**Comment by keanjapesan** [comment]
recheck

**Comment by robimarko** [comment]
CentOS7 is being used for whatever reason instead of Debian.
@jwagantall Any ideas?

**Comment by keanjapesan** [comment]
@sonoble @robimarko @taraschornyiplv the job configuration was set with a CentOS node, no idea how it was working before, so the solution is to change the node name from ci-management repo to "prd-ubuntu-docker-aws-1c-2g".
Yesterday, for testing purposes I was running some rechecks with the Ubuntu image and we found some troubles on the first attempts until we finally got it working. However the permanent fix is still pending so please push a PR modifying the following line https://github.com/dentproject/ci-management/blob/master/jjb/dentos/dentos.yaml#L10 with "prd-ubuntu-docker-aws-1c-2g"

**Review by snobleaws — DISMISSED** [review]

File: packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch [hunk 1]
```diff
@@ -0,0 +1,51 @@
+From 5c5d857711d80c3750c61103bfbaa547fa0e3f56 Mon Sep 17 00:00:00 2001
+From: Taras Chornyi <taras.chornyi@plvision.eu>
+Date: Mon, 1 Nov 2021 12:20:10 +0200
+Subject: [PATCH] net: phylink: sfp: Add quirk for FINISAR 10G modules
+
+Finisar FTLF8536P4BCL and FTLX1471D3BCL can operate
+in 1000Base-x mode as well
```
Comment (paulmenzel) [inline packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch:7]: Please reflow for 75 characters per line.

**Review by paulmenzel — CHANGES_REQUESTED** [review]
Have you upstreamed this already? Are you going to upstream this?

File: packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch (hunk 1 above)
Comment (Mickey201) [reply packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch:7]: will be fixed.
We will also up-streamed it soon.

**Comment by Mickey201** [comment]
Applying Paul comment.
Now with GPG signoff. @sonoble , @taraschornyiplv , please review.

**Review by taraschornyiplv — APPROVED** [review]

**Comment by Mickey201** [comment]
@keanjapesan,  the build failed again for the same reason: 
23:48:03 + sudo apt-get install -y binfmt-support
23:48:03 sudo: apt-get: command not found

**Comment by taraschornyiplv** [comment]
recheck

**Comment by snobleaws** [comment]
The failure appears to be an issue with the patch?

17:56:29 Applying: 0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch
17:56:29 /w/workspace/dentos-verify-main/packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch: line 1: From: command not found
17:56:29 /w/workspace/dentos-verify-main/packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch: line 2: syntax error near unexpected token `newline'
17:56:29 /w/workspace/dentos-verify-main/packages/base/any/kernels/5.10-lts/patches/0042-net-phylink-sfp-Add-quirk-for-FINISAR-10G-modules.patch: line 2: `From: Taras Chornyi

**Comment by Mickey201** [comment]
Closing this pull request.
Something went wrong with the pushed files.
Will open a new one that will include all resolved comments.

**Comment by paulmenzel** [comment]
For completeness, the new merge/pull request is https://github.com/dentproject/dentOS/pull/150.

@Mickey201, it’d be great, if you did that the next time. I know it’s frustrating that QA/CI/… fails, but for people not working on this eight hours a day, it’s very helpful.

**Pull request closed without merging**